homeassistant/components/denonavr/receiver.py

```diff
@@ -85,5 +87,7 @@ async def async_init_receiver_class(self) -> None:
         # Use httpx.AsyncClient getter provided by Home Assistant
         receiver.set_async_client_getter(self._async_client_getter)
         await receiver.async_setup()
+        if self._use_telnet:
+            await receiver.async_telnet_connect()
```

Comment:
In combination with my previous comment suggesting not running the entire update when telnet is connected, we should update the receiver before connecting telnet, that all values are available from the beginning.
```suggestion
            await receiver.async_update()
            await receiver.async_telnet_connect()
```

Reply:
Thinking we'd also want to call `async_update_audyssey` here too if the user has that configured?

Reply:
Resolved in https://github.com/home-assistant/core/pull/85980/commits/b5f983ec35264389e2eb7e3417e31d6a208ec700 along with Audyssey update